src/main/java/com/google/devtools/build/lib/bazel/bzlmod/GitRepoSpecBuilder.java

```diff
@@ -86,11 +86,6 @@ public GitRepoSpecBuilder setPatchCmds(List<String> patchCmds) {
     return setAttr("patch_cmds", patchCmds);
   }
 
-  @CanIgnoreReturnValue
-  public GitRepoSpecBuilder setPatchCmdsWin(List<String> patchCmdsWin) {
-    return setAttr("patch_cmds_win", patchCmdsWin);
```

Comment:
Nit: this could be useful for Windows users to avoid Bash dependency on Windows. But it seems it was actually never exposed.

Reply:
Yes, I agree that it could be useful (also on the other overrides). I mostly wanted to remove it so that it is clear that it's missing.

Reply:
SG!